package.json

```diff
@@ -67,5 +67,7 @@
     "dev": "web-dev-server --app-index index.html  --root-dir dist --node-resolve --open",
     "start": "yarn build-assets && concurrently --kill-others --names js,css,dev-server 'yarn watch' 'yarn build-css --watch' 'yarn dev'"
   },
-  "dependencies": {}
+  "dependencies": {
+    "webdriverio": "^7.19.5"
+  }
```

Comment:
Shouldn't this be in `devDependencies`? Trix now pulls in a whole bunch of extra packages that don't seem necessary at all.

Reply:
👍 fixed in https://github.com/basecamp/trix/pull/1091

Reply:
Thanks!